### DISCUSSION TITLE
No module named ' pyexcel'

### DISCUSSION THREAD

**Discussion by MarloesAlb** [the-anylogic-company/AnyLogic-Pypeline#33] [Q&A] [answered]
Hi there, I am trying to use the pyCommunicator to retrieve data from a python file. In my python file, I have imported pyexcel, when I run the python code everything works.
Now I try running the python file by calling it in anylogic via:
pyCommunicator.run("import GAresults");
So, GAresults is my python file. However, anylogic gives the error ' no module named ' pyexcel''
But I have imported it in the python file. 
Do you know how I can solve this?

**Comment by t-wolfeadam** [comment, marked as answer]
Likely what's happening is the version of Python you used to run the file directly is different than the one being referenced inside AL. 

If you click on the pycom object during a running model, it'll show the path to the executable its using.

And for running it directly, you can check by adding this to your script file: 

```python
import sys
print(sys.executable)
```

(If you're using a virtual environment, you may need to use `os.__file__` instead)

**Reply by t-wolfeadam** [reply]
And if they are different, you'll need to setup the pycom object appropriately

**Reply by t-wolfeadam** [reply]
(I'll just assume this is the case)
